pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1184,10 +1171,16 @@ public static void updateKit(String dsmKitId, Shipment participantShipment, Ship
         if (results.resultException != null) {
             logger.error("Error updating kit w/ dsm_kit_id " + dsmKitId, results.resultException);
         } else {
-            logger.info("Updated kit w/ dsm_kit_id " + dsmKitId, results.resultException);
+            logger.info("Updated kit w/ dsm_kit_id " + dsmKitId);
+            upsertUpdatedKitInfoIntoES(ddpInstanceDto, kitRequestShipping, dsmKitId);
+        }
+    }
 
+    private static void upsertUpdatedKitInfoIntoES(DDPInstanceDto ddpInstanceDto, KitRequestShipping kitRequestShipping, Object dsmKitId) {
+        if (ddpInstanceDto.getEsParticipantIndex() != null) {
```

Comment:
@denniscunningham this prevents updates for Juniper

Reply:
Right. As we discussed it would be nice to create a more explicit way of expressing that.